src/core/EdFi.DataManagementService.Core.Tests.Unit/Middleware/ValidateDocumentMiddlewareTests.cs

```diff
@@ -190,6 +190,52 @@ public void It_should_not_have_response()
         }
     }
 
+    [TestFixture]
+    public class Given_A_Request_With_Overposted_Object_Property : ValidateDocumentMiddlewareTests
+    {
+        private PipelineContext _context = No.PipelineContext();
+
+        [SetUp]
+        public async Task Setup()
+        {
+            string jsonData =
+                """{"schoolId": 989, "gradeLevels":{"gradeLevelDescriptor": "grade1", "gradeLevelOverPost": "overPostedValue"},"nameOfInstitution":"school12", "objectOverpost": { "x": "overPostedValue"}}""";
```

Comment:
Should you include this `gradeLevelOverPost` in your tests below? I don't see it mentioned. You could also just do a test of what the `ParsedBody.ToJsonString()` _should_ actually be.